feature-libs/product-configurator/rulebased/components/add-to-cart-button/configurator-add-to-cart-button.component.spec.ts

```diff
@@ -691,16 +706,7 @@ describe('ConfigAddToCartButtonComponent', () => {
     });
   });
 
-  describe('displayOnlyButton', () => {
```

Comment:
Why is this test deleted? As far as I see it it would reveal an issue because the navigation to overview from checkout and back is still not working

Reply:
The test has been moved to the `leaveConfigurationOverview` section